pkg/kubelet/dockertools/manager.go

```diff
@@ -579,6 +580,16 @@ func (dm *DockerManager) runContainer(
 		// of CPU shares.
 		cpuShares = milliCPUToShares(cpuRequest.MilliValue())
 	}
+	var devices []dockercontainer.DeviceMapping
+	if nvidiaGPULimit.Value() != 0 {
+		// Experimental. For now, we hardcode /dev/nvidia0 no matter what the user asks for
+		// (we only support one device per node).
```

Comment:
What happens in docker if I have privileged and I want to use hostPath to map in the whole /dev directory?  Does that work?